site-src/concepts/api-overview.md

```diff
@@ -133,91 +150,94 @@ different relationships that Gateways and Routes can have:
   a single Route to control application exposure simultaneously across different
   IPs, load balancers, or networks.
 
-*In summary, Gateways select Routes and Routes control their exposure. When a
-Gateway selects a Route that allows itself to be exposed, then the Route will
-bind to the Gateway. When Routes are bound to a Gateway it means their
-collective routing rules are configured on the underlying load balancers or
-proxies that are managed by that Gateway. Thus, a Gateway is a logical
+*In summary, Routes attach to Gateways and Gateways choose which attachments to
+allow. When a Route tries to attach to a Gateway that does not prevent it, then
+the Route will bind to the Gateway. When Routes are bound to a Gateway it means
+their collective routing rules are configured on the underlying load balancers
+or proxies that are managed by that Gateway. Thus, a Gateway is a logical
 representation of a networking data plane that can be configured through
 Routes.*
```

Comment:
I'm not sure what happened here, but I'd actually already updated a good chunk of the Route->Gateway attachment section for v1alpha2 when this page was versioned (https://github.com/kubernetes-sigs/gateway-api/pull/762/files#diff-bb2765a3020ed8962e5e749402a8ace4539c6218bdc621a98625ca19b2700e82R89). I think a lot of what you've done here is better than my initial work, but it may be helpful to merge some of the concepts together here.

I remember in that PR, every way I tried to reword this still sounded confusing to me. I think it may be better to remove or simplify this paragraph in favor of further explanation below.